### PR TITLE
Bplan without officeworker account

### PR DESCRIPTION
**Pull request by pallix**

### CODE REVIEW

**Comment by pallix** [comment]
Branch does not build because the frontend needs a little change @xi @MagdaN

File: src/adhocracy_meinberlin/adhocracy_meinberlin/sheets/test_bplan.py [hunk 1]
```diff
@@ -102,8 +102,7 @@ def test_create(self, meta, context):
     @mark.usefixtures('integration')
     def test_get_empty(self, meta, context, registry):
         inst = meta.sheet_class(meta, context)
-        wanted = {'office_worker': None,
-                  'plan_number': '',
+        wanted = {'plan_number': '',
                   'participation_kind': '',
```
Comment (joka) [inline src/adhocracy_meinberlin/adhocracy_meinberlin/sheets/test_bplan.py]: Why is office_worker_email missing here?

**Comment by joka** [comment]
backend lgtm

**Comment by pallix** [comment]
@MagdaN can you review frontend?

**Comment by joka** [comment]
@magda @xi can we merge this?

**Comment by pallix** [comment]
@MagdaN can you review the code?

**Pull request merged**